webapp/channels/src/actions/invite_actions.ts

```diff
@@ -167,10 +168,10 @@ export function sendGuestsInvites(
         const results = await Promise.all(users.map((user) => sendGuestInviteForUser(dispatch, user, teamId, channels, members)));
 
         for (const result of results) {
-            if (result.sent) {
+            if ('sent' in result) {
                 sent.push(result.sent);
             }
-            if (result.notSent) {
+            if ('notSent' in result) {
```

Comment:
Being purist in terms of typing, `sent` can be in result, and still be `undefined`, so we only want to send it when it is, and it is defined.

0/5 on whether this affect this logic in any way.

Reply:
Good catch. I'll change that. I went with `in` because it made TS happy, but this is one of those areas where making TS happy can break other things